internal/cmd/commands/targetscmd/funcs.go

```diff
@@ -383,6 +430,17 @@ func extraFlagsHandlingFuncImpl(c *Command, _ *base.FlagSets, opts *[]targets.Op
 		if len(c.flagHostId) != 0 {
 			*opts = append(*opts, targets.WithHostId(c.flagHostId))
 		}
+
+	case "set-static-address":
+		if len(c.flagAddress) == 0 {
+			c.UI.Error("No Target static address supplied. Please supply it using -address.")
+			return false
+		}
+		if c.flagAddress == "null" {
+			*opts = append(*opts, targets.DefaultAddress())
+		} else {
```

Comment:
I do really dislike this but I think you already made the case that this is consistent with other flags?